public/js/hummingbird-treeview.js

```diff
@@ -0,0 +1,1569 @@
+		    //add items to dropdown
+		    $(dialog + " #" + search_output).append('<li id="drop_' + $(this).attr("id")  + '"><a href="#">' + $(this).parent().text() + '</a></li>');
+		    num_search_results++;
+		}
+	    });
+	    if (num_search_results == 0) {
+		$(dialog + " #" + search_output).append("&nbsp; &nbsp; Nothing found");
+	    }	    
+	    //click on search dropdown
+	    $(dialog + " #" + search_output + " li").on("click", function(e) {
+		//no focus on the input field to trigger the search scrolling
+		e.preventDefault();
+	    	//hide dropdown
+	    	$(dialog + " #" + search_output).hide();
+	    	//set value of input field
+	    	$(dialog + " #" + search_input).val($(this).text());
+	    	//reset color of last selection
+	    	if (first_search == false) {
+		    if (this_var_checkbox.prop("disabled")) {
+			this_var_checkbox.parent("label").parent("li").css({'color':'#c8c8c8',"cursor":"not-allowed"});
+		    } else {
+			this_var_checkbox.parent("label").parent("li").css({'color':'black',"cursor":"pointer"});
+		    }
+	    	}
+	    	//before jumping to the hummingbird-end-node a collapse all is needed
+	    	tree.hummingbird("collapseAll");
+	    	//get this checkbox
+	    	this_var_checkbox = tree.find('input[id="' + $(this).attr("id").substring(5) + '"]');
+		//parent uls
+	    	var prev_uls = this_var_checkbox.parents("ul");
+		//change plus to minus
+	    	prev_uls.closest("li").children("i").removeClass(collapsedSymbol).addClass(expandedSymbol);
+		//highlight hummingbird-end-node
+	    	this_var_checkbox.parent("label").parent("li").css({'color':'#f0ad4e'});
+	    	first_search = false;
+	    	//expand parent uls
+		prev_uls.show();
+		//---------------------------scrolling-----------------------------------//
+		//set scroll position to zero
+		if (treeview_container == "body") {
+		    //Chrome
+		    document.body.scrollTop = 0;
+		    //Firefox
+		    document.documentElement.scrollTop = 0;
+		} else {		    
+		    $(dialog + " #" + treeview_container)[0].scrollTop = 0;
+		}		    
+		//get position and offset of element
+		var this_var_checkbox_position = this_var_checkbox.position().top;
+		this_var_checkbox_position = this_var_checkbox_position+scrollOffset;
+		
+		if (treeview_container == "body") {
+		    //Chrome
+		    document.body.scrollTop += this_var_checkbox_position;
+		    //Firefox
+		    document.documentElement.scrollTop += this_var_checkbox_position;
+		} else {
+		    $(dialog + " #" + treeview_container)[0].scrollTop = this_var_checkbox_position;
+		}		    
+		//---------------------------scrolling-----------------------------------//
+	    });
+	    //if there is only one search result -> go to this without showing the dropdown
+	    if (num_search_results == 1) {
+	    	var one_search_id = $("#" + search_output + " li").attr("id");
+	    	$("#" + one_search_id).trigger("click");
+	    }
+	    
+	});
+    }
+    //----------------------------search--------------------------------------//
+})(jQuery);
```

Comment:
Is there a minified version of this that we can use?

Reply:
We will see.

Reply:
Hi @brichardssa we have found the solution for this issue and now implementing.